hardware-linux.json

```diff
@@ -0,0 +1,80 @@
+{
+    "display": {
+        "control": {
+            "enabled": false
+        }
+    },
+    "proximity": {
+        "enabled": false
+    },
+    "batteryCharger": {
+        "enabled": false
+    },
+    "batteryFuelGauge": {
+        "enabled": false
+    },
+    "buttonInterruptHandler": {
+        "enabled": false
+    },
+    "wiringPi": {
+        "enabled": false
+    },
+    "hapticMotor": {
+        "enabled": false
+    },
+    "systemservice": {
+        "enabled": true,
+        "systemd": {
+            "sudo" : false,
+            "timeout" : 30000,
+            "services": {
+                "wifi": "wpa_supplicant@wlan0.service",
+                "dns": "systemd-resolved.service",
+                "webserver": "lighttpd.service",
+                "ntp": "systemd-timesyncd.service",
+                "dhcp": "dhcpcd.service",
+                "shutdown": "shutdown.service",
+                "yio-update": "update.service",
+                "zeroconf": "avahi-daemon",
+                "networking": "systemd-networkd"
+            }
+        }
+    },
+    "wifi": {
+        "enabled": true,
+        "maxScanResults": 20,
+        "joinRetryCount": 5,
+        "joinRetryDelay": 3000,
+        "pollInterval": 20000,
+        "useShellScript": false,
+        "interface": {
+            "wpa_supplicant" : {
+                "socketPath": "/var/run/wpa_supplicant/wlan0",
+                "removeNetworksBeforeJoin": false
+            },
+            "shellScript": {
+                "sudo": false,
+                "timeout": 30000,
+                "clearNetworks": "",
+                "connect": "/usr/bin/yio-remote/wifi_network_create.sh",
+                "listNetworks": "/usr/bin/yio-remote/wifi_network_list.sh",
+                "startAP": "/usr/bin/yio-remote/reset-wifi.sh",
+                "getSSID": "/usr/bin/yio-remote/wifi_ssid.sh",
+                "getIP": "/usr/bin/yio-remote/wifi_ip.sh",
+                "getMAC": "cat /sys/class/net/wlan0/address",
+                "getRSSI": "/usr/bin/yio-remote/wifi_rssi.sh"
+            }
+        },
+        "accessPoint": {
+            "TODO": true
+        }
+    },
+    "webserver" : {
+        "enabled": true,
+        "lighttpd" : {
+            "configFile": "/etc/lighttpd/lighttpd.conf",
+            "wifiSetupConfig": "/etc/lighttpd/lighttpd-wifisetup.conf",
+            "webConfiguratorConfig": "/etc/lighttpd/lighttpd-config.conf"
```

Comment:
These won't be necessary as we discussed to have the initial setup on the remote itself. So we will only server one website (the configurator).

Reply:
This needs a bigger cleanup later on, also in remote-os.
We should probably keep the AP functionality somehow as last-resort recovery. But let's discuss that outside of this PR

Reply:
Sure!